data/transition-sites/gds_alphagov_alert.yml

```diff
@@ -0,0 +1,10 @@
+---
+site: gds_alphagov_alert
+whitehall_slug: government-digital-service
+homepage: https://alert.publishing.service.gov.uk/
```

Comment:
@jennyd: Could you see more about why that is? I've done this on a branch:
https://github.com/alphagov/transition-config/compare/remove_trailing_slashes?expand=1